### PR TITLE
Request screen printer is loading faster than the page is rendering

### PR DESCRIPTION
**Pull request by marcoAntonioNina**
Fixed [FOUR-2522](https://processmaker.atlassian.net/browse/FOUR-2522)

add waiting time when loading the print page

### CODE REVIEW

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-2522-a-processmaker.ci.processmaker.net

File: resources/js/requests/components/screenDetail.vue [hunk 1]
```diff
@@ -68,13 +73,34 @@
...
+          }
+        }, 10000);
+      });
     },
     methods: {
+      closeRequestCount() {
+        window.ProcessMaker.apiClient.requestCount = 0;
+        window.ProcessMaker.apiClient.requestCountFlag = false;
+      },
+      checkComponentsPrint() {
```
Comment (danloa) [inline resources/js/requests/components/screenDetail.vue:97]: Use a explicit name for the function, like "printWhenNoRequestsArePending"

File: resources/js/app-layout.js [hunk 2]
```diff
@@ -215,6 +224,10 @@ window.ProcessMaker.apiClient.interceptors.response.use((response) => {
   // response.config.method (PUT, POST, DELETE)
   // response.config.url (extract resource name)
   window.ProcessMaker.EventBus.$emit("api-client-done", response);
+  //flags print forms
+  if (window.ProcessMaker.apiClient.requestCountFlag) {
+    window.ProcessMaker.apiClient.requestCount--;
```
Comment (danloa) [inline resources/js/app-layout.js:229]: It looks like requestCount can have negative numbers. To avoid problems do not decrease it if it is 0

**Review by danloa — CHANGES_REQUESTED** [review]

**Review by danloa — APPROVED** [review]

**Review by boliviacoca — APPROVED** [review]

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-2522-a-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://bugfix-four-2522-a-processmaker.ci.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://develop-processmaker.ci.processmaker.net

**Pull request merged**